lib/Common/Memory/RecyclerWriteBarrierManager.cpp

```diff
@@ -218,6 +226,10 @@ X64WriteBarrierCardTableManager::Initialize()
 {
     AutoCriticalSection critSec(&_cardTableInitCriticalSection);
 
+#if ENABLE_DEBUG_CONFIG_OPTIONS
+    RecyclerWriteBarrierManager::Initialize();
+#endif
+
```

Comment:
Is there any potential concurrent concern? If so maybe move this after _cardTable initialized?